mouse/record.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
```

Comment:
Also, add a module docstring. This can use features for [reStructuredText]( http://docutils.sourceforge.net/rst.html ). Simple example with proper format can be found here ( http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html ).